resources/views/volumes/clone.blade.php

```diff
@@ -0,0 +1,195 @@
+                            <div class="form-group{{ $errors->has('clone_annotations') ? ' has-error' : '' }}">
+                                @if($errors->has('clone_annotations'))
+                                    <span class="help-block">{{ $errors->first('clone_annotations') }}</span>
+                                @endif
+                            </div>
+
+                            <div v-if="cloneAnnotationLabels">
+                                <div>
+                                    <label>
+                                        <input type="checkbox" v-if="cloneAnnotationLabels" id="annotationLabel"
+                                               v-model="restrictAnnotationLabels"> Restrict annotation labels (<span
+                                            v-text="selectedAnnotationLabelsCount"></span> labels selected)
+                                    </label>
+                                </div>
+                                <label-trees v-if="cloneAnnotationLabels && restrictAnnotationLabels"
+                                             :trees="annotationLabelTrees"
+                                             :multiselect="true"
+                                             :allow-select-siblings="true" :allow-select-children="true"
+                                             class="request-labels-well well well-sm"></label-trees>
+                            </div>
+                            <div class="form-group{{ $errors->has('only_annotation_labels') ? ' has-error' : '' }}">
+                                @if($errors->has('only_annotation_labels'))
+                                    <span class="help-block">{{ $errors->first('only_annotation_labels') }}</span>
+                                @endif
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <input type="submit" class="btn btn-success pull-right" :disabled="cannotSubmit" value="Clone">
+                    <a class="btn btn-default pull-right" :disabled="loading" type="button"
+                       href="{{\Illuminate\Support\Facades\URL::previous()}}">Cancel</a>
+                    <input v-for="id in annotationLabelIds" type="hidden" name="only_annotation_labels[]"
+                           v-bind:value="id">
+                    <input v-for="id in fileLabelIds" type="hidden" name="only_file_labels[]" v-bind:value="id">
+                    <input v-for="file in selectedFiles" type="hidden" name="only_files[]" v-bind:value="file.id">
+                    <input v-for="file in selectedFiles" type="hidden" name="selected_files[]"
+                           v-bind:value="file.filename">
+                    <input type="hidden" name="clone_annotations" v-if="cloneAnnotationLabels" v-bind:value=1>
+                    <input type="hidden" name="clone_annotations" v-else v-bind:value=0>
```

Comment:
There should be no need for this. You should be able to directly utilize the visible checkbox:

```html
<input type="checkbox" id="annotations" name="clone_annotations" v-model="cloneAnnotationLabels">
```

If this doesn't work, we should discuss it in our meeting.